downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/DatabaseDumps.java

```diff
@@ -0,0 +1,53 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.ProcessBuilder.Redirect;
+import java.util.Arrays;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+public class DatabaseDumps {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void execute(MySQLAdaptor dba, int releaseNumber, String username, String password, String host, int port, String testReactomeDatabase) throws IOException, InterruptedException {
+		// Take mysqldumps of 'stable_identifiers' and 'test_reactome_##' and compress them using gzip.
+		logger.info("Generating DatabaseDumps");
+		// Create databases folder that will hold the DB dumps
+		Runtime.getRuntime().exec("mkdir -p " + releaseNumber + "/databases");
+		
+		// Start with stable_identifiers dump
+		//TODO: Sticking to GK naming conventions?
+		logger.info("Dumping stable_identifiers to gk_stable_ids.sql...");
+		Runtime.getRuntime().exec("touch " + releaseNumber + "/databases/gk_stable_ids.sql");
+		File stableIdsFile = new File(releaseNumber + "/databases/gk_stable_ids.sql");
+		String[] stableIdsCommand = new String[]{"mysqldump", "-h" + host,"-u" + username, "-p" + password, "-P" + port, "stable_identifiers"};
+		ProcessBuilder stableIdsprocessBuilder = new ProcessBuilder(Arrays.asList(stableIdsCommand));
+		stableIdsprocessBuilder.redirectError(Redirect.INHERIT);
+		stableIdsprocessBuilder.redirectOutput(Redirect.to(stableIdsFile));
+		Process stableIdsProcess = stableIdsprocessBuilder.start();
+		stableIdsProcess.waitFor();
+		
+		Process gzipStableIds  = Runtime.getRuntime().exec("gzip " + releaseNumber + "/databases/gk_stable_ids.sql");
```

Comment:
I think Java has a library for gzip functionality (https://docs.oracle.com/javase/8/docs/api/java/util/zip/GZIPOutputStream.html).